docs/preparations.rst

```diff
@@ -257,6 +356,6 @@ What about Sumo?
 
 Odds are that you are implementing rich metadata export so that you can start utilizing
 Sumo. Producing metadata with exported data is a pre-requisite for using Sumo. When you
-have undertaken the steps above, you are good to go! Head to the 
-`Sumo <https://fmu-sumo.app.radix.equinor.com/documentation>`__ documentation to
+have undertaken the steps above, you are good to go! Head to 
+`Sumo <https://fmu-sumo.app.radix.equinor.com/>`__ and click "documentation" to
```

Comment:
```suggestion
`Sumo <https://fmu-sumo.app.radix.equinor.com/>`_ and click "documentation" to
```
I think the double underscore is typically used for footnotes. Maybe it is here too, not sure